### PR TITLE
how-to: optimize pd operation and upgrade monitor version

### PR DESCRIPTION
**Pull request by liubo0127**
### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with:
- [ ] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

**Comment by liubo0127** [comment]
@pcqz PTAL

File: how-to/monitor/monitor-a-cluster.md [hunk 1]
```diff
@@ -289,20 +289,22 @@ url = https://grafana.net
     - 默认账户：admin
     - 默认密码：admin
 
-2. 点击 Grafana 图标打开侧边栏。
+    > 注意：
```
Comment (anotherrachel) [inline how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > **注意：**
```

File: how-to/monitor/monitor-a-cluster.md [hunk 2]
```diff
@@ -289,20 +289,22 @@ url = https://grafana.net
...
+    >
+    > **Change Password** 步骤可以选择 **Skip**
```
Comment (anotherrachel) [inline how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > **Change Password** 步骤可以选择 **Skip**。
```

File: how-to/scale/with-ansible.md [hunk 3]
```diff
@@ -224,28 +224,33 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
...
 
     {{< copyable "shell-regular" >}}
 
     ```bash
-    ansible-playbook rolling_update.yml
+    ansible-playbook start.yml -l 172.16.10.103
     ```
 
-6. 启动监控服务：
+    > 注意：
```
Comment (anotherrachel) [inline how-to/scale/with-ansible.md]:
```suggestion
    > **注意：**
```

File: how-to/scale/with-ansible.md [hunk 4]
```diff
@@ -224,28 +224,33 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
...
+    >
+    > 如果使用了别名（inventory_name），则也需要使用 `-l` 指定别名。
```
Comment (anotherrachel) [inline how-to/scale/with-ansible.md]:
```suggestion
    > 如果使用了别名 (inventory_name)，则也需要使用 `-l` 指定别名。
```

File: how-to/scale/with-ansible.md [hunk 5]
```diff
@@ -458,6 +463,10 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
     ansible-playbook stop.yml -l 172.16.10.2
     ```
 
+    > 注意：
```
Comment (anotherrachel) [inline how-to/scale/with-ansible.md]:
```suggestion
    > **注意：**
```

**Comment by anotherrachel** [comment]
@pcqz PTAL, thanks!

**Comment by pcqz** [comment]
LGTM

**Pull request merged**